dev/playbooks/roles/fedora-common-packages/tasks/main.yml

```diff
@@ -1,5 +1,5 @@
 - name: update packages cache
-  yum: update_cache=yes
+  yum: name=yum update_cache=yes
```

Comment:
You can do something like `action: "{{ ansible_pkg_mgr }} update_cache=yes"`. That will call yum or dnf depending on the available command, on Fedora 23 and 24 that should run just DNF.